core/connection.go

```diff
@@ -65,6 +84,8 @@ func CreateConnection(src, dst *ProvableChain, to time.Duration) error {
 }
 
 func createConnectionStep(src, dst *ProvableChain) (*RelayMsgs, error) {
+	zapLogger := logger.GetLogger()
+	defer zapLogger.Zap.Sync()
```

Comment:
Logging in `createConnectionStep` is important. Same as `createChannelStep`.
The following points must be logged as info logs.
- What heights are used to query the current states of connections?
- Which switch case is selected?

Reply:
1. channel
https://github.com/dongrie/yui-relayer/blob/add-logger/core/channel.go#L119
https://github.com/dongrie/yui-relayer/blob/add-logger/core/channel.go#L184-L199
2. connection
https://github.com/dongrie/yui-relayer/blob/add-logger/core/connection.go#L149
https://github.com/dongrie/yui-relayer/blob/add-logger/core/connection.go#L228-L244

I have log output at each step like this, but is this not enough?